spec/models/organization_spec.rb

```diff
@@ -33,10 +33,6 @@
       Dictionaries.currencies << 'EUR' unless Dictionaries.currencies.include?('EUR')
     end
 
-    after do
-      Dictionaries.currencies.delete('EUR') if Dictionaries.currencies.include?('EUR')
```

Comment:
@yunixon I think it's wrong fix. In this spec we need to use some currency that we don't have implemented in rates. Use AED (Emirati Dirham) for example.